doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -949,6 +949,7 @@ Documentation
   * The :ref:`asset_tracker_template_redirect` page, which provides the information about the `Asset Tracker Template Add-on <Asset Tracker Template_>`_.
   * The :ref:`log_rpc` library documentation page.
   * The :ref:`mcuboot_serial_recovery` documentation page, based on the official Zephyr documentation, which discusses the implementation and usage of the serial recovery.
+  * The :ref:`ug_hpf_softperipherals_comparison` documentation page, describing potential use cases and differences between both solutions.
```

Comment:
Shouldn't this be rather in the Developing with coprocessors section?

Reply:
added